### DISCUSSION TITLE
김영명님이 2025 academic conference에 새롭게 합류합니다.

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#438] [General]
과거 Academic Conference 멤버였던 차동훈님의 직장 동료인 @ymkim97 김영명님이 참여 의사를 밝혀 주셨습니다.
참여 신청 날짜가 2025 Academic Conference Orientation이 있던 2024-12-27일이었어서 그날은 참석 못하고
첫 리뷰 모임 부터 참석할 예정입니다.

따라서 2025 Academic Conference의 스타트 멤버는 총 10명이 됩니다.

**Comment by ymkim97** [comment]
안녕하세요 이번에 새롭게 2025 academic conference 합류하게 된 김영명입니다!
오리엔테이션은 참여하지 못하였지만 첫 리뷰 모임 때 다시 한번 인사 드리겠습니다.
앞으로 잘 부탁드립니다 여러분!!!

**Reply by ymkim97** [reply]
@jongfeel 
안녕하세요!
지금 PR을 위해서 push를 하려고 하는데 권한이 필요합니다! 혹시 권한 부여 가능할까요!?

**Reply by jongfeel** [reply]
제가 organization 초대를 해 드리고 권한을 안드렸었습니다.

지금 권한 설정 해서, 다시 해보시면 될 겁니다.

**Reply by ymkim97** [reply]
감사합니다 저녁에 확인하고 다시 해보겠습니다!
